tests/Integration/WebTestCase.php

```diff
@@ -94,7 +94,7 @@ protected function tearDown()
         return Kernel::class;
     }
 
-    private function refreshContainer()
+    protected function refreshContainer()
```

Comment:
This change and the change in `ExceptionListenerTest` appear to be necessary, but I have to admit they *smell* a bit. Not sure of how that could be improved at the moment, but maybe we can take a look some other time.

Reply:
I agree.